src/components/carousel/Carousel.tsx

```diff
@@ -1,15 +1,19 @@
-import React, { useContext } from 'react';
+import React from 'react';
 
-import { ConfigContext } from '../../context';
+import { useColors } from '../../contexts/colors';
```

Comment:
I know that this isn't most important in "this case", but "for production" probably I would introduce aliases to avoid dots hell.